src/scripts/main.js

```diff
@@ -1,3 +1,61 @@
 'use strict';
 
-// write code here
+const thead = document.querySelector('thead');
+
+function sort(nameSort) {
+  const tbody = document.querySelector('tbody');
+  const tr = tbody.querySelectorAll('tr');
+  let n;
+
+  switch (nameSort) {
+    case 'Name':
+      n = 0;
+      break;
+    case 'Position':
+      n = 1;
+      break;
+    case 'Age':
+      n = 2;
+      break;
+    case 'Salary':
+      n = 3;
+      break;
+  }
+
+  const people = [...tr].sort((a, b) => {
+    if (n === 0 || n === 1) {
+      return (a.querySelectorAll('td')[n].textContent).localeCompare(
+        b.querySelectorAll('td')[n].textContent);
+    }
+
+    return (
+      a.querySelectorAll('td')[n].textContent.replace('$', '').replace(',', ''))
+    - b.querySelectorAll('td')[n].textContent.replace('$', '').replace(',', '');
+  });
+
+  const tbodyNew = document.createElement('tbody');
+
+  tbodyNew.innerHTML = `
+    ${people.map(human => `
+    <tr>
+    <td>${human.querySelectorAll('td')[0].innerText}</td>
+    <td>${human.querySelectorAll('td')[1].innerText}</td>
+    <td>${human.querySelectorAll('td')[2].innerText}</td>
+    <td>${human.querySelectorAll('td')[3].innerText}</td>
```

Comment:
It's better to move all repeated values to a variables. `human.querySelectorAll('td')` for example